transcom/mymove
# MB-16012 loadtest verify

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16012)

## Summary

Run the load test as an integration test, but do not require it to pass

### How to test

This is a CI only change, so check out this PR and confirm that failures do not prevent merging

NOTE: This PR should only be merged after https://github.com/transcom/mymove/pull/10628
